programming_examples/basic/dma_transpose/aie2.py

```diff
@@ -59,7 +59,6 @@ def sequence(A, B, C):
                 npu_dma_memcpy_nd(
                     metadata="in", bd_id=1, mem=A, sizes=[1, K, M, 1], strides=[1, 1, K]
                 )
-                npu_sync(column=0, row=0, direction=0, channel=0)
```

Comment:
I'd prefer that the tests keep the sync explicit, but using `aiex.npu.dma_wait` instead of `aiex.npu.sync`

Reply:
> I'd prefer that the tests keep the sync explicit, but using `aiex.npu.dma_wait` instead of `aiex.npu.sync`

Should we close this PR in favor of https://github.com/Xilinx/mlir-aie/pull/1791 ?  Or is there still a desire to insert the sync/wait automatically?